lib/dialects/oracle/index.js

```diff
@@ -80,10 +79,10 @@ Object.assign(Client_Oracle.prototype, {
   // Get a raw connection, called by the `pool` whenever a new
   // connection needs to be added to the pool.
   acquireRawConnection() {
-    return new Bluebird((resolver, rejecter) => {
+    return new Promise((resolver, rejecter) => {
       this.driver.connect(this.connectionSettings, (err, connection) => {
         if (err) return rejecter(err);
-        Bluebird.promisifyAll(connection);
+        connection.executeAsync = promisify(connection.execute);
```

Comment:
here attention needed: check all file and notice, what from connection only execute (with suffix async) is used

Reply:
FYI: I discovered earlier today that `lib/dialects/oracle` is actually partially-dead code.  It is not used anywhere except as a parent class for `lib/dialects/oracledb`.

So, you might want to double-check that this method is even used by `lib/dialects/oracledb`.  If not, then it might be easiest to just remove the entire method.

Reply:
It's easier to drop support for https://www.npmjs.com/package/oracle entirely, but this is not logical part of this merge request